src/test/librados_test_stub/TestMemIoCtxImpl.cc

```diff
@@ -209,9 +210,20 @@ int TestMemIoCtxImpl::omap_get_vals(const std::string& oid,
     }
     ++it;
   }
+  if (pmore) {
+    *pmore = (it != omap.end());
+  }
```

Comment:
missing `*pmore = false;` in early return above where `o_it == m_pool->file_omaps.end()`